pkgs/applications/misc/far2l/default.nix

```diff
@@ -79,7 +69,7 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   meta = with stdenv.lib; {
-    description = "An orthodox file manager";
+    description = "Linux port of FAR Manager v2 (http://farmanager.com/)";
     homepage = "https://github.com/elfmz/far2l";
     license = licenses.gpl2;
```

Comment:
Please update the license to either `licenses.gpl2Only` or `licenses.gpl2Plus`: https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206